extra/mariabackup/backup_mysql.cc

```diff
@@ -126,7 +126,7 @@ xb_mysql_connect()
 	mysql_options(connection, MYSQL_OPT_PROTOCOL, &opt_protocol);
 	mysql_options(connection,MYSQL_SET_CHARSET_NAME, "utf8");
 
-	msg("Connecting to MySQL server host: %s, user: %s, password: %s, "
```

Comment:
Changes in this file are not necessary for purpose of this  MDEV.

Reply:
yep sure.

Thanks for the review.